vitrina/datasets/views.py

```diff
@@ -734,6 +735,8 @@ def get_form_class(self):
 
         if subclass.name == DCATResourceSubclass.SERVICE:
             return ServiceResourceForm
+        if subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Comment:
Optional (for readability)
```suggestion
        elif subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Reply:
Not fixing. Same reason as above.